subscription-engine/kafka-ws/README.md

```diff
@@ -316,9 +343,25 @@ If you are developing, you can use `nodemon` too:
 ```shell
 npm run dev
 ```
-## NOTE
-To use WebSocket with Nginx, Kong, Api gateway or similar, look for timeout settings. In Nginx, for example: proxy_connect_timeout, proxy_send_timeout and proxy_connect_timeout; And in Kong on a Service, for example: connect_timeout, write_timeout and read_timeout.
+__NOTE THAT_ in order to use WebSocket with nginx, Kong, API gateway or similar, look for timeout
+settings. Examples:
+- nginx: `proxy_connect_timeout`, `proxy_send_timeout` and
+`proxy_connect_timeout`
+- Kong: connect_timeout, write_timeout and read_timeout.
 
-## **Examples**
+# **Examples**
 
 Check the [examples](./examples) directory for more info.
+
+# **Documentation**
+
+Check the documentation for more information:
+
+- [Latest Kafka WS API documentation](https://dojot.github.io/kafka-ws/apiary_latest.html#kafka-ws)
+- [Development Kafka WS API documentation](https://dojot.github.io/kafka-ws/apiary_development.html#kafka-ws)
+- [Latest dojot platform documentation](https://dojotdocs.readthedocs.io/en/latest)
+
+# **Issues and help**
```

Comment:
I don't know if we need put **Issues and help** here, because it isn't the main readme, but i think this is a not a problem too.